### DISCUSSION TITLE
Only a very limited number of emails shown

### DISCUSSION THREAD

**Discussion by crides** [pimalaya/himalaya#378] [General]
Hi, I was looking for a TUI/CLI email program and stumbled on this after seen `neomutt`. However, as with `neomutt`, `himalaya` only shows 850 emails, when I have ~15k in total. I can't really find what these have in common, but they are like somehow selected from here and there.. This is for my school email, and it seems like they use Office365 Outlook as the backend. Any ideas?

**Comment by soywod** [comment]
I'm not sure to understand your problem. You mean that `neomutt` and `himalaya` only display 850 emails at maximum? Did you try in other MUA like Thunderbird?

**Reply by crides** [reply]
Sorry, you can ignore this. It turns out to be a very weird thing with how my school handled email backends..

**Reply by soywod** [reply]
Really strange :sweat_smile: sorry not to be able to help you. Do not hesitate if you have any further questions!
